README.md

```diff
@@ -0,0 +1,12 @@
+# Javascript :
+
+        In JS today I learn some concepts and practice on it and use some concepts in js problems.
+        - foreach
+        - map
+        - filter
+        - reduce
+
+# React :
+
+     I've started to learn react from today.
```

Comment:
You can add these in description of PR and you can directly send an update to Ricky over slack